docs/uperf.md

```diff
@@ -18,19 +18,24 @@ metadata:
   name: uperf-benchmark
   namespace: ripsaw
 spec:
-  name: uperf
-  args:
-    hostnetwork: false
-    pin: true
-    pin_server: "master-0"
-    pin_client: "master-1"
-    # Server size must always be 1 or more
-    pairs: 1
-    proto: tcp
-    test_type: stream
-    nthr: 2
-    size: 16384
-    runtime: 60
+  workload:
+    # cleanup: true
+    name: uperf
+    args:
+      hostnetwork: false
+      pin: false
+      pin_server: "master-0"
+      pin_client: "master-1"
+      rerun: 1
```

Comment:
Do you document what `rerun` means?

Reply:
Oh, good point I do not. I will add that!